courses/migrations/0029_revert_certificates_prior_aug_8.py

```diff
@@ -0,0 +1,129 @@
+# Generated by Django 3.2.15 on 2022-10-26 15:40
+
+from django.db import migrations
+from datetime import datetime, timezone
+from cms.models import CertificatePage
+from courses.models import CourseRunCertificate, ProgramCertificate
+
+
+AUGUST_8_2022 = datetime(2022, 8, 8, tzinfo=timezone.utc)
+SEPTEMBER_20_2022 = datetime(2022, 9, 20, tzinfo=timezone.utc)
```

Comment:
August 8 is the date we want to revert the revisions for

September 20 is the date when the [original data migration](https://github.com/mitodl/mitxpro/pull/2416/files#diff-4fd4c50e37874ef380afc22f90ca12071425b20ba1e08c166ab99d130953d02eR1) ran